proto/greenfield/sp/query.proto

```diff
@@ -39,7 +38,12 @@ service Query {
 
   // Queries a storage provider with specify address
   rpc StorageProvider(QueryStorageProviderRequest) returns (QueryStorageProviderResponse) {
-    option (google.api.http).get = "/greenfield/storage_provider/{spAddress}";
+    option (google.api.http).get = "/greenfield/storage_provider/{id}";
+  }
+
+  // Queries a list of StorageProviderByOperatorAddress items.
```

Comment:
The response seems not be a list of storage providers.

Reply:
Change it to `Queries a StorageProvider by specify operator address.`